src/markov/markov_approx.jl

```diff
@@ -416,6 +420,27 @@ function discrete_var(b::Union{Real, AbstractVector},
     return MarkovChain(P, [X[:, i] for i in 1:Nm^M])
 end
 
+"""
+check persistency when `method` is `Quadrature` and give warning if needed
+
+##### Arguments
+- `B::Union{Real, AbstractMatrix}` : impact coefficient
+- `method::VAREstimationMethod` : method for grid making
+
+##### Returns
+- `nothing`
+
+"""
+function warn_persistency(B::AbstractMatrix, ::Quadrature)
+    if any(eig(B)[1] .> 0.9)
+        warn("The quadrature method may perform poorly for highly persistent processes.")
+    end
+    return nothing
+end
+warn_persistency(B::Real, method::Quadrature) = warn_persistency(fill(B, 1, 1), method)
+warn_persistency(::Union{Real, AbstractMatrix}, ::Union{Even, Quantile}) = nothing
```

Comment:
Instead of `::Union{Even, Quantile}`, let's have the second argument be `::VAREstimationMethod` so  that is the default for any additional `VAREstimationMethod` subtypes we might create in the future.